zcash_client_backend/Cargo.toml

```diff
@@ -23,11 +23,15 @@ jubjub = "0.5.1"
 nom = "5.1.2"
 protobuf = "2.20"
 rand_core = "0.5.1"
+ripemd160 = { version = "0.9.1", optional = true }
+secp256k1 = { version = "0.19", optional = true }
+sha2 = "0.9"
```

Comment:
should this dependency also be optional? Perhaps it is also associated with the [same feature flag](https://github.com/zcash/librustzcash/pull/341/files#r577069716) used for `hdwallet`? It appears to only be used in keys.rs when deriving a transparent address.